lib/rules/no-return-await.js

```diff
@@ -44,7 +46,19 @@ module.exports = {
             context.report({
                 node: context.getSourceCode().getFirstToken(node),
                 loc: node.loc,
-                messageId: "redundantUseOfAwait"
+                messageId: "redundantUseOfAwait",
+                suggest: [
+                    {
+                        messageId: "removeAwait",
+                        data: { type: "block" },
```

Comment:
```suggestion
```

`data` is used to to populate [placeholders](https://eslint.org/docs/latest/developer-guide/working-with-rules#placeholders-in-suggestion-messages), but this message doesn't have any.

Reply:
Thanks!

Reply:
I've fixed this